start_cerberus.py

```diff
@@ -111,6 +112,24 @@ def main(cfg):
                              "to True and assuming that the nodes are ready")
                 watch_nodes_status = True
 
+            # Monitor cluster operators status
+            if watch_cluster_operators:
+                operators_status = runcommand.invoke("kubectl get co -o yaml")
+                status_yaml = yaml.load(operators_status, Loader=yaml.FullLoader)
+                watch_cluster_operators_status, failed_operators = \
```

Comment:
We need to use watch_cluster_operators_status as well to determine final cerberus_status ( cerberus_status = watch_nodes_status and watch_namespaces_status )